data/applications/mlpipelines.yaml

```diff
@@ -0,0 +1,17 @@
+metadata:
+  name: mlpipelines
+  annotations:
+    opendatahub.io/categories: 'ML Pipelines'
+spec:
+  displayName: ML Pipelines
+  provider: Kubeflow
+  description: |-
+    Pipeline tool specialized for Machine Learning workloads, leveraging Kubeflow and Openshift Pipelines
```

Comment:
The descriptions of most other apps are focused on what a customer can use them for rather than a definition. What can ML Pipelines help the user achieve? Why do they want this thing?

Would something like the following work?

"ML Pipelines is a continuous integration and deployment tool that helps you automate model deployment in a portable, scalable way."

Reply:
I worry that leading off with `a continuous integration and deployment tool` wouldn't appropriately signify the focus around ML Operations, and more seems like a pure DevOps tool, which may be a bit misleading.

This is a bit of a step away from the KFP text, but perhaps something along the lines of 

> ML Pipelines is a workflow platform with a focus on enabling Machine Learning operations such as Model development, experimentation, orchestration and automation.

would work?  Although the list of 'capabilities' is a bit lengthy for a one-liner so we could slim that down a bit, too.